src/main/java/org/spongepowered/api/stats/Statistic.java

```diff
@@ -40,13 +39,11 @@
     String getId();
 
     /**
-     * Gets the {@link StatisticFormat} of this statistic. If this is not
-     * present that this statistic's format is deferred to its group's default
-     * format.
+     * Gets the {@link StatisticFormat} of this statistic.
      *
-     * @return The format of this statistic, if available
+     * @return The format of this statistic
      */
-    Optional<StatisticFormat> getStatisticFormat();
+    StatisticFormat getStatisticFormat();
```

Comment:
Sorry for keeping on discussion, but i really want to talk this through.
Lets try it the other way round.

- What does the old method tell us about the group.
  - IMO: Nothing, am i wrong here? My version states whether the group has a format and if it has one it can be used for all contained Statistic.
- What does the old method tell us about the statistic with the group.
  - Whether it has the same format as the group. Well my version does that too.
- Does the old method prevent adding statistics to the wrong group?
  - No, you can add any `Statistic` `TIMES_KICKED` to `StatisticGroup.ITEM_USED`. (IMO the API should prevent that). My version does prevent that.

These are my concerns regarding the current system. Could you please explain it (1+3) to me then i won't bother you anymore. Sorry for the extra work this causes you.